.github/workflows/pipeline.yml

```
@@ -0,0 +1,22 @@
+---
+name: Pipeline
+on: [push]
+jobs:
+  tests:
+    name: Unit Tests
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        ruby: [2.6, 2.7, 3.0, 3.1]
```

Due to a bug with YAML parsing `3.0` needs to be wrapped in quotes so that it doesn't turn into a float and get rounded to be just `3`. In this project it doesn't make a large difference because the unit tests all still seem to be passing, but this will test on the latest release of `ruby 3` rather than the latest patch of ruby `3.0`.
```suggestion
        ruby: [2.6, 2.7, '3.0', 3.1]
```